src/Tizen.Nlp/Tizen.Nlp/NaturalLanguageProcess.cs

```diff
@@ -0,0 +1,372 @@
+        /// <since_tizen> 5 </since_tizen>
+        public NaturalLanguageProcess()
+        {
+            _tag = Application.Current.ApplicationInfo.ApplicationId;
+            _noti.Received += ResultReceived;
+        }
+
+
+        /// <summary>
+        /// An async method  to connect remote service.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        /// <exception cref="InvalidOperationException">Thrown when the connect is rejected.</exception>
+        public Task Connect()
+        {
+            if (_isConnected)
+            {
+                throw new InvalidOperationException("message already connected");
+            }
+
+            if (_isConnecting)
+            {
+                throw new InvalidOperationException("message is connecting");
+            }
+            else
+            {
+                _isConnecting = true;
+            }
+
+            TaskCompletionSource<bool> tcs = new TaskCompletionSource<bool>();
+            _msg = new Message(ServiceId);
+            _msg.Disconnected += (s, e) =>
+            {
+                _isConnected = false;
+                _isConnecting = false;
+                Disconnected?.Invoke(this, e);
+            };
+            _msg.Connected += (sender, e) =>
+            {
+                Log.Debug(_tag, "start to register");
+                _msg.CoRegister(Application.Current.ApplicationInfo.ApplicationId, _noti);
+                tcs.SetResult(true);
+                _isConnected = true;
+                _isConnecting = false;
+            };
+            _msg.Rejected += (sender, e) =>
+            {
+                tcs.SetException(new InvalidOperationException("invalid id cause exception"));
+            };
+            _msg.Connect();
+            return tcs.Task;
+        }
+
+        /// <summary>
+        /// A method to close message connection
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public void Close()
+        {
+            if (!_isConnected) return;
+            _msg.UnRegister();
+            _msg.Dispose();
+            _isConnected = false;
+            _msg = null;
+            _isConnecting = false;
+        }
+
+
+        private void MapClear()
+        {
+            _mapsWordTokenize.Clear();
+            _mapsPosTag.Clear();
+            _mapsNamedEntity.Clear();
+            _mapsLemmatize.Clear();
+            _mapsLangDetect.Clear();
+        }
+
+        /// <summary>
+        /// A method to release resource of library
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposed)
+                return;
+
+            if (disposing)
+            {
+                Close();
+                _noti.Received -= ResultReceived;
+                _noti = null;
```

Comment:
_noti is not released, 
You have to call _noti.Dispose();

Reply:
sorry, no _noti.Dispose()  method in proxy side